source/components/molecules/index.ts

```diff
@@ -11,9 +11,10 @@ export { default as StoryWrapper } from './StoryWrapper';
 export { default as HelpButton } from './HelpButton';
 export { default as EditableList } from './EditableList';
 export { default as CheckboxField } from './CheckboxField';
-export { default as Modal } from './Modal';
```

Comment:
Varför ändrar vi exporten här? Känns som att det bryter mot resterande exporter?

Reply:
I think because the Modal folder exports two things, Modal and the hook useModal, so it doesn't have a default export, and thus didn't work as written? Feel free to check that, but I suspect that's why I changed it.